rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,228 @@
+              level: extended
+              type: keyword
+              description: >
+                Difference Hash (dhash) to find files with a visually similar icon or thumbnail.
+              example: b806e17c8e330d82
+
+    - name: debug
+      level: extended
+      type: nested
+      description: Debug information, if present
+      fields:
+        - name: offset
+          level: extended
+          type: keyword
+          description: Debug offset information.
+          example: 1296336
+
+        - name: size
+          level: extended
+          type: long
+          format: bytes
+          description: Size of the debug information.
+          example: 816
+
+        - name: type_str
+          level: extended
+          type: keyword
+          description: Format of the debug information.
+          example: IMAGE_DEBUG_TYPE_POGO
+
+        - name: type
+          level: extended
+          type: keyword
+          description: Information type generated by the debug options.
+          example: 13
+
+        - name: timestamp
+          level: extended
+          type: date
+          description: Timestamp of the debug information.
+          example: "2020-11-05T17:25:47.000Z"
+
+    - name: import_list
+      level: extended
+      type: keyword
+      description: List of all imported functions
+      example: { "library_name" : "mscoree.dll", "imported_functions" : "GetFileVersionInfoSizeA" }
+
+    - name: sections
+      level: extended
+      description: >
+        Data about sections of compiled binary PE
+      type: nested
+      fields:
+        - name: chi2
+          level: extended
+          description: Chi-square probability distribution.
+          type: long
+          example: 3027194
+
+        - name: virtual_address
+          level: extended
+          description: Virtual address available to the file.
+          type: long
+          format: bytes
+          example: 8192
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the file.
+          type: float
+          example: 6.24
+
+        - name: flags
+          level: extended
+          description: Section flags of the file.
+          type: keyword
+          example: rx
+
+        - name: name
+          level: extended
+          description: Section names of the file.
+          type: keyword
+          example: .text, .data
+
+        - name: raw_size
+          level: extended
+          description: Size of the section or the dize of the initialized data on disk.
+          type: long
+          format: bytes
+          example: 198144
+
+    - name: resource_details
+      level: extended
+      type: keyword
+      description: >
+        If the PE contains resources, some info about them
+      example: { "chi2": -1, "filetype": "English text", "entropy": 0, "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855", "lang": "CHINESE SIMPLIFIED" }
```

Comment:
What about making this `nested` and explode it into structured fields (also rename `resources`)?

```
resources.chi2
resources.filetype
resources.entropy
resources.hash.sha256
resources.language
resources.type <-- this is new, it's what gets rolled up below (i.e. RT_VERSION, etc.)
```

Reply:
Done